src/api/general-services/pipeline-manage/constructors/create-new-job-if-not-exist.js

```diff
@@ -16,6 +16,7 @@ const createNewJobIfNotExist = (context, step) => {
         Payload: {
           image: pipelineImages['remoter-server'],
           name: 'pipeline-remoter-server',
+          detached: true,
```

Comment:
what is this for?

Reply:
detached is used by the docker image as a flag to start in detached mode or not.

`remoter-server` can be started as daemon, hence it's set to true. `remoter-client` has to be started in non-detached mode so that it runs sequentially as defined in the step functions definition.

Reply:
would the user be the one that decides if the container should be running in the background (i.e. in detached mode)? In which cases in particular will we want to run in detached mode?

Reply:
No, this setting is set internally, not by the user. As I understand it, it's to enforce sequential creation and execution of the `remoter-client` containers, so the steps are executed in order.

Reply:
I see, thanks for explaining!